home_assistant_datasets/tool/leaderboard/prebuild.py

```diff
@@ -56,16 +56,20 @@ def write_eval_output(cmds: list[str], output_file: pathlib.Path) -> None:
 
     def build_report(eval_report: EvalReport) -> None:
         print(f"Generating report for outputs in {eval_report.directory}")
-        cmds = EVAL_CMD + [
-            "--output_type=report",
-            f"--model_output_dir={eval_report.directory}",
-        ]
-        write_eval_output(cmds, eval_report.report_file)
-        cmds = EVAL_CMD + [
-            "--output_type=csv",
-            f"--model_output_dir={eval_report.directory}",
-        ]
-        write_eval_output(cmds, eval_report.csv_file)
+        if eval_report.directory.parent.name == 'automations':
+            # TODO: Implement
+            ...
+        else:
+            cmds = EVAL_CMD + [
+                "--output_type=report",
+                f"--model_output_dir={eval_report.directory}",
+            ]
+            write_eval_output(cmds, eval_report.report_file)
+            cmds = EVAL_CMD + [
+                "--output_type=csv",
+                f"--model_output_dir={eval_report.directory}",
+            ]
+            write_eval_output(cmds, eval_report.csv_file)
```

Comment:
The reports for automations are written by its `eval` command now in a single step. We can update `assist` to work that way later. For now, lets just continue here.

```suggestion
        if eval_report.directory.parent.name == 'automations':
            continue
        cmds = EVAL_CMD + [
            "--output_type=report",
            f"--model_output_dir={eval_report.directory}",
        ]
        write_eval_output(cmds, eval_report.report_file)
        cmds = EVAL_CMD + [
            "--output_type=csv",
            f"--model_output_dir={eval_report.directory}",
        ]
        write_eval_output(cmds, eval_report.csv_file)
```